lib/watir/element_collection.rb

```diff
@@ -80,12 +80,38 @@ def last
     #
 
     def to_a
-      # TODO: optimize - lazy element_class instance?
       @to_a ||= elements.map.with_index do |e, idx|
         element_class.new(@query_scope, @selector.merge(element: e, index: idx))
       end
     end
 
+    #
+    # Returns true if two element collections are equal.
+    #
+    # @example
+    #   browser.select_list(name: "new_user_languages").options == browser.select_list(id: "new_user_languages").options
+    #   #=> true
+    #
+
+    def ==(other)
+      to_a == other.to_a
+    end
+    alias_method :eql?, :==
+
+    def +(other)
```

Comment:
For `#siblings`. It's essentially `element.previous_siblings + element.following_siblings`, so there needed to be a plus operator.